sampleWorkspace/rover/src/test.js

```diff
@@ -0,0 +1,30 @@
+import gql from "graphql-tag";
+
+sdfsdfs;
+gql`
+  """
+  The query type, represents all of the entry points into our object graph
+  """
+  type Query {
+    me: User!
+  }
+
+  """
+  Test
+  """
+  type User {
+    id: ID!
+    name: String!
+  }
+`;
+
+console.log("foobar!");
```

Comment:
Same as above - this file (as everything in sampleWorkspace) is only meant to open as an "example workspace" so we can see where highlighting works.

If the whole `.js` file would consist out of `gql` tags, it would be pointless - there has to be some filler. But the file will never be executed, it purely exists to look at it in the editor.